src/signals/incident-management/containers/IncidentOverviewPage/reducer.js

```diff
@@ -35,6 +38,16 @@ export default (state = initialState, action) => {
         .set('errorMessage', action.payload)
         .set('loading', false);
 
+    case SEARCH_INCIDENTS:
+      return state
+        .set('loading', true)
+        .set('error', false)
+        .set('errorMessage', undefined)
+        .set('searchQuery', action.payload);
+
+    case RESET_SEARCH_INCIDENTS:
+      return state.set('searchQuery', '');
```

Comment:
Should this not also set `loading` back to `false`?

Reply:
Or does this not cancel any requests?

Reply:
Doesn't cancel any requests, no. It clears a part of the state that is used in, for instance, the `PageHeader` container component to show what the user searched for. Setting it to an empty string will clear the `PageHeader` title.